### PR TITLE
Add entry in update.lst for latest guntamatic binding changes

### PR DESCRIPTION
**Pull request by MikeTheTux**
Add entry in update.lst for latest guntamatic binding changes

openhab/openhab-addons#17901

### CODE REVIEW

**Comment by lsiepel** [comment]
Related PR is merged

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -157,6 +157,9 @@ ALERT;MQTT Binding (Home Assistant): Thing types and channel IDs have been signi
 ALERT;MQTT Binding (Home Assistant): Legacy schema vacuums are no longer supported.
 ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been organized into groups. You will need to reconfigure your setup to the new thing structure.
 
+[5.0.0]
+ALERT;Guntamatic Binding: The Binding now uses channel groups and the dynamic status channels now have an index prefix to allow multiple channels with the same description. Re-connect changed and new channels to items.
```
Comment (lsiepel) [inline distributions/openhab/src/main/resources/bin/update.lst:161]:
```suggestion
ALERT;Guntamatic Binding: The Binding now uses channel groups and the dynamic status channels now have an index prefix to allow multiple channels with the same description. Existing Items will need to be adjusted.
```

**Review by lsiepel — APPROVED** [review]

**Review by holgerfriedrich — APPROVED** [review]

**Review by holgerfriedrich — APPROVED** [review]
LGTM, thanks!

**Review by holgerfriedrich — CHANGES_REQUESTED** [review]
@MikeTheTux could you pls. change the signoff (real-name policy)? I think you have done this for the commit in add-ons repo. Thanks!

**Review by holgerfriedrich — APPROVED** [review]
Don't mind, lets consider this as a small patch, not requiring this.

**Pull request merged**